www/src/pages/components/close-button.mdx

```diff
@@ -0,0 +1,53 @@
+import { graphql } from 'gatsby';
+
+import ComponentApi from '../../components/ComponentApi';
+import ReactPlayground from '../../components/ReactPlayground';
+import CloseButtonBasic from '../../examples/CloseButton/Basic';
+import CloseButtonDisabled from '../../examples/CloseButton/Disabled';
+import CloseButtonVariants from '../../examples/CloseButton/Variants';
+import CloseButtonLabelled from '../../examples/CloseButton/Labelled';
+
+
+# Close Button
+
+A generic close button for dismissing content such as modals and alerts.
+
+<ReactPlayground codeText={CloseButtonBasic} />
+
+## Disabled state
+
+Bootstrap adds relevant styling to a disabled close button to prevent user
+interactions.
+
+<ReactPlayground codeText={CloseButtonDisabled} />
+
+## Variants
+
+All standard visual variants are available for both animation styles by
+setting the `variant` property. Alternatively spinners can be custom
+sized with the `style` property, or custom CSS classes.
```

Comment:
```suggestion
Change the default dark color to white using `variant="white"`.
```
Just added a random blurb to replace the spinner text.  You can add more if needed.